packages/shared/src/components/squads/SteppedDetails.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FormEventHandler, ReactElement, useContext } from 'react';
+import { Modal } from '../modals/common/Modal';
+import { ModalState, SquadStateProps } from './utils';
+import { ModalPropsContext } from '../modals/common/types';
+import { SquadDetails } from './Details';
+import { SquadForm } from '../../graphql/squads';
+
+export function SteppedSquadDetails({
```

Comment:
Nice lovely to see we can re-use this one as well 🤯

Reply:
Yeap, thanks to your work, it was easy to reuse all these